setup.py

```diff
@@ -88,6 +88,7 @@
         "console_scripts": [
             "frag_duck=scripts.run_full_from_fragalysis:main",
             "get_wqb=scripts.get_wqb:main",
+            "duck_prepare_sys_for_amber=scripts.duck_prepare_sys_for_amber:main",
```

Comment:
Adding duck_prepare_sys_for_amber here so it's on `PATH`, once the list of files in the script directory has been sorted out, we could add the rest here.